docs/faq/obs-setup.md

```
@@ -0,0 +1,91 @@
+---
+title: OBS Setup FAQ
+description: Common questions about configuring overlays in OBS Studio.
+---
+
+# OBS Setup
+
+Common questions about adding overlays to OBS and optimizing performance.
+
+### What browser source settings should I use?
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)